packages/app/src/elements/item-view.ts

```diff
@@ -632,49 +588,46 @@ export class ItemView extends Routing(StateMixin(LitElement)) {
-                                                ${$l(
-                                                    "Will expire {0} day(s) after the last update",
-                                                    this._expiresEveryPeriodOfDays?.toString() || "0"
-                                                )}.
-                                            </strong>
-                                        </div>
-                                          <pl-input
-                                              id="expiresEveryPeriodOfDaysInput"
-                                              class="small margined"
-                                              .placeholder=${$l("Enter Item Expiration in Days")}
-                                              type="number"
-                                              pattern="[0-9]*"
-                                              @input=${() => {
-                                                  this._expiresEveryPeriodOfDays = Number.parseInt(
-                                                      this._expiresEveryPeriodOfDaysInput.value,
-                                                      10
-                                                  );
-                                              }}
-                                              .value=${this._expiresEveryPeriodOfDays?.toString() || "0"}
-                                              select-on-focus
+                                              class="small padded centering horizontal layout border-bottom border-top"
                                           >
-                                          </pl-input>
+                                              ${$l("Expire")}
+                                              <pl-input
+                                                  id="expiresAfter"
+                                                  class="slim margined text-right-aligning"
+                                                  type="number"
+                                                  pattern="[0-9]*"
+                                                  @input=${() => {
+                                                      this._expiresAfter = Number.parseInt(
+                                                          this._expiresAfterInput.value,
+                                                          10
+                                                      );
+                                                  }}
+                                                  .value=${this._expiresAfter?.toString() || ""}
+                                                  select-on-focus
+                                                  style="width: 4em"
+                                              >
+                                              </pl-input>
+                                              ${$l("days after being updated.")}
+                                          </div>
```

Comment:
This will make translations a bit harder, but I think it's worth it for the cleaner look.